### PR TITLE
Demo: Start API in debug mode by default

### PR DESCRIPTION
**Pull request by abel-castro**
This PR sets the local dev server to start in debug mode by default for the demo/api. So now when you run `npm run dev` the api will start in debug mode.

### CODE REVIEW

**Comment by nsams** [comment]
dev-pm is running start:dev: https://github.com/vivid-planet/comet/blob/main/dev-pm.config.js#L204C22-L204C64

(we should not have 4 different start scripts)

File: demo/api/package.json [hunk 1]
```diff
@@ -20,8 +20,7 @@
         "mikro-orm": "mikro-orm",
         "mikro-orm:drop": "mikro-orm schema:drop -r",
         "mikro-orm:migration:generate": "mikro-orm migration:create",
-        "start": "$npm_execpath prebuild && dotenv -c -- nest start",
-        "start:debug": "$npm_execpath prebuild && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
+        "start": "$npm_execpath prebuild && dotenv -c -- nest start --debug --watch --preserveWatchOutput",
```
Comment (johnnyomair) [inline demo/api/package.json:23]: You'll need to add this to start:dev.

**Comment by johnnyomair** [comment]
> (we should not have 4 different start scripts)

IMO we should only have `dev` and `serve` (for prod)

File: demo/api/package.json (hunk 1 above)
Comment (abel-castro) [reply demo/api/package.json:23]: Added in [last commit ](https://github.com/vivid-planet/comet/pull/2532/commits/09a4e93d22b7623c5c9e1819853e4b611934dbad)

**Comment by nsams** [comment]
> IMO we should only have dev and serve (for prod)

and they should be unified across all microservices (now we have start:dev, dev and start :laughing:)

**Review by nsams — APPROVED** [review]

**Pull request merged**